Gameplay/entities/player/PlayerCamera.cpp

```diff
@@ -78,27 +78,55 @@ void Hachiko::Scripting::PlayerCamera::SetLookAhead()
 
 void Hachiko::Scripting::PlayerCamera::MoveCamera()
 {
-	const math::float2 mouse_movement_x_z = MoveCameraWithMouse();
-	const math::float3 mouse_movement = float3(mouse_movement_x_z.x, 0.0f, mouse_movement_x_z.y);
-	ScrollWheelZoom(&_relative_position_to_player);
+	if (Input::IsGamepadModeOn())
+	{
+		const math::float2 gamepad_normalized_position =
+			math::float2(Input::GetAxisNormalized(Input::GameControllerAxis::CONTROLLER_AXIS_RIGHTX), Input::GetAxisNormalized(Input::GameControllerAxis::CONTROLLER_AXIS_RIGHTY));
+		const math::float3 controller_movement = float3(gamepad_normalized_position.x * 4.0f, 0.0f, gamepad_normalized_position.y * 4.0f);
+		ScrollWheelZoom(&_relative_position_to_player);
```

Comment:
Do we need to have check for zoom here as it's gamepad mode?

Reply:
I don't know if it does something important to set the camera so I leaved it :/